components/patient-data/impl/src/main/java/org/phenotips/data/internal/controller/GeneListController.java

```diff
@@ -116,7 +118,7 @@
     private static final String JSON_REJECTEDGENES_KEY = "rejectedGenes";
 
     private static final List<String> STATUS_VALUES = Arrays.asList(INTERNAL_CANDIDATE_VALUE, INTERNAL_REJECTED_VALUE,
-        INTERNAL_SOLVED_VALUE);
+        INTERNAL_SOLVED_VALUE, INTERNAL_CARRIER_VALUE);
```

Comment:
This comment has nothing to do with the 'carrier' issue, but hardcoding property values in the controller is wrong and, as we can see here, difficult to maintain. The value should be loaded from the XClass.